src/services/addressResolver/index.ts

```diff
@@ -25,6 +25,7 @@ interface EntityLookupProps {
   endpoint: string;
   apiHeader?: string;
   apiKey?: string;
+  path: string | undefined;
```

Comment:
by the time this api has been called path should be there already, if not can don't call this api liao. Don't make sense to check for it this late.